src/store/setToolMode.js

```diff
@@ -56,12 +63,39 @@ const setToolActiveForElement = function(
         options[`is${interactionType}Active`] = false;
       }
     });
+
+    if (
+      globalConfiguration.state.showSVGCursors &&
+      tool.supportedInteractionTypes.includes('Mouse')
+    ) {
+      _setToolCursorIfPrimary(element, options, tool);
+    }
   }
 
   // Resume normal behavior
   setToolModeForElement('active', null, element, toolName, options);
 };
 
+function _setToolCursorIfPrimary(element, options, tool) {
+  let mouseButtonMask;
+
+  if (typeof options === 'number') {
+    mouseButtonMask = options;
+  } else {
+    mouseButtonMask = options.mouseButtonMask;
+  }
```

Comment:
I think @mikehazell's introduced a method that normalizes the options higher up the chain, so we can check this in a consistent way.

IE. `options.mouseButtonMask` will always be an array after being normalized.

Reply:
@dannyrb That change hasn't been merged (#903) , also the normalisation happens inside `setToolModeForElement` which is called after this method.

Reply:
And in this case, there's no guarantee that the `options.mouseButtonMask` being passed in is an array either.

Reply:
Thanks for casting options too an array if its just a number Mike. Can `options.mouseButtonMask` be anything other than an array within the library?